superset-frontend/src/dashboard/components/PropertiesModal/index.tsx

```diff
@@ -656,6 +657,23 @@ const PropertiesModal = ({
                   {t(
                     'This JSON object is generated dynamically when clicking the save or overwrite button in the dashboard view. It is exposed here for reference and for power users who may want to alter specific parameters.',
                   )}
+                  {onlyApply && (
+                    <>
+                      {' '}
+                      {t(
+                        'Please DO NOT overwrite `filter_scopes` metadata.',
```

Comment:
```suggestion
                        'Please DO NOT overwrite the "filter_scopes" key.',
```

I wonder if `"filter_scopes"` is clearer given it's how the key is presented. I'm less concerned about the key vs metadata name.